script.js

```diff
@@ -14,3 +70,490 @@ backMenu.forEach((element) => element.addEventListener('click', () => {
+                  <li>Web Development</li>
+               </ul>
+           </li>
+     </ul>
+     
+  </div>
+  </div>
+  `;
+  body.appendChild(about);
+
+  /*  about section
+      and
+    get resume section above
+*/
+
+  // ----contact section below -----
+
+  contact.innerHTML = `
+  <div class="contact-info">
+      <p>
+          I'm always <br> interested in 
+          hearing  <br> about new projects, 
+          <br> so if you'd like to <br> chat please 
+          get in  <br> touch.
+      </p>
+  </div>
+  <form class="contact-form" action="https://formspree.io/f/mdovbjlz" method="post">
+      <div class="form-group">
+          <input class="input" type="text" name="name" maxlength="30" placeholder="Full name" rows="10" required>
+          <input class="input" id="email" type="email" name="email" placeholder="Email address" required>
+          <textarea name="send" maxlength="399" required>Write me something</textarea>
+          <input class="submit" type="submit" value="Get in touch">
+          <span id="error_message"></span>
+      </div>
+  </form>
+  
+  `;
+  body.appendChild(contact);
+  // ----contact section below -----
+
+  hr.innerHTML = `
+    <hr class="hr">
+  `;
+  body.appendChild(hr);
+  // ----footer section below -----
+  footer.innerHTML = `
+  <div class="icon">
+      <ul class="nav-menu">
+          <li><a href="https://github.com/BlessmanNewton"><i class="fa fa-github" aria-hidden="true"></i></a></li>
+          <li><a href="https://linkedin.com/BlessmanNewton"><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
+          <li><a href="https://angellist.com/BlessmanNewton"><i class="fa fa-angellist" aria-hidden="true"></i></a></li>
+          <li><a href="https://twitter.com/BlessmanNewton"><i class="fa fa-twitter" aria-hidden="true"></i></i></a></li>
+          <li><a href="https://medium.com/BlessmanNewton"><i class="fa fa-medium" aria-hidden="true"></i></a></li>
+      </ul>
+  </div>
+  <hr class="hr_footer">
+  `;
+
+  body.appendChild(footer);
+  // ----contact section above -----
+
+  const view = document.querySelectorAll('.view-project');
+  view.forEach((element) => element.addEventListener('click', () => {
+    const show = document.createElement('div');
+    const detailPop = document.createElement('div');
+    show.className = 'show';
+    detailPop.className = 'pop';
+    detailPop.innerHTML = `
+                      <div class="popup-header">
+                        <h2 class="desktop">Keeping track of hundreds
+                         of components website</h2>
+                         <h2 class="mob-title">Multi Post Stories</h2>
+                        <span id="close">&times;</span>
+                      </div>
+                        <div class="popup-button">
+                          <button id="lists">HTML</button>
+                          <button id="lists">Boostrap</button>
+                          <button id="lists">Ruby on rails</button>
```

Comment:
Please kindly adjust your design so that the buttons look similar to the given template

Your design
![image](https://github.com/Blessman-Newton/Microverse_Portfolio/assets/78251927/801b7f5f-3609-4681-b42c-4c2d12119e3f)

Figma
![image](https://github.com/Blessman-Newton/Microverse_Portfolio/assets/78251927/9f02cd46-0088-4e44-9ef4-372580b9eeb3)